ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml

```
@@ -122,6 +138,31 @@ resources:
       memory: 4000Mi
 test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
 tests:
+- as: eks-qa-e2e-tests
+  optional: true
```

These will all need to be postsubmit jobs to run against the release branch.
```suggestion
  postsubmit: true
```